tendrl/gluster_integration/sds_sync/cluster_status.py

```diff
@@ -59,7 +59,7 @@ def sync_cluster_status(volumes):
         peer_count=peer_count,
         vol_count=len(volumes),
         volume_up_degraded=degraded_count
-    ).save()
+    ).save(sync_ttl)
```

Comment:
pls pass as keyword argument

Reply:
Ah yes. Will do that.

Reply:
Are you making this change? @shtripat

Reply:
Ah yes. Looks like didnt push the changes. will do that now